README.md

```diff
@@ -927,6 +929,27 @@ Returns the device's type as a string, which will be one of:
 const deviceType = DeviceInfo.getDeviceType(); // 'Handset'
 ```
 
+### isAutoDateAndTime()
+
+Tells if the Get Automatic date & time setting is enabled in phone.
+
+**Examples**
+
+```js
+const isAutoDateAndTime = DeviceInfo.isAutoDateAndTime(); // true
+```
+
+### isAutoTimeZone()
+
+Tells if the Get Automatic time zone setting is enabled in phone.
+
+**Examples**
+
+```js
+const isAutoTimeZone = DeviceInfo.isAutoTimeZone(); // true
```

Comment:
I'm really sorry I didn't see this before - the usage example should be for promise - it should end with .then(etc) - there are other examples for other Promise APIs.

I swear this is the last thing :-), this looks good to merge if the example is updated - thanks for the continued effort + patience

Reply:
Not a problem, actually my mistake haven't seen those little things properly. Any how its all sorted now. Thanks for pointing out the mistakes with patience.